libs/core/src/ports/port.ts

```diff
@@ -57,6 +57,10 @@ const disposeAndExit = async (
   // don't kill process when errors are caught in production
   if (code === 'ERROR' && config.NODE_ENV === 'production' && !forceExit)
     return;
+  else if (code === 'ERROR' && config.NODE_ENV === 'production' && forceExit) {
+    // sleep for 1 second to allow stat metrics to flush
+    await new Promise((resolve) => setTimeout(resolve, 1_000));
+  }
```

Comment:
```
if(code==='ERROR' && config.NODE_ENV==='production') {
   if(forceExit) await new Promise....
   else return;
}
```
reads better

Reply:
we also have the delay util to avoid rewriting the same promise again and again

Reply:
yeah I was gonna say the syntax is a little odd even if it's valid